frontend/src/api/backend.ts

```diff
@@ -3,5 +3,5 @@ import axios from 'axios';
 axios.defaults.withCredentials = true;
 
 export default axios.create({
-    baseURL:"http://127.0.0.1:8000"
+  baseURL: process.env.REACT_APP_BACKEND_URL,
```

Comment:
Since you are adding new variable on the `.env` file, update the `.env.example` file